src/openrct2/world/Footpath.h

```diff
@@ -139,6 +139,18 @@ enum
     FOOTPATH_CORNER_3 = (1 << 3),
 };
 
+enum
+{
+    FOOTPATH_CONNECTION_SE_SW = (1 << 0),
```

Comment:
Since these flags are used for path elements directly north/east/south/west of the paths (and not SE _and_ SW for example), I'd rather see them named like:

```suggestion
    FOOTPATH_CONNECTION_S = (1 << 0),
```

Reply:
Diagonal could be seen differently depending on your perspective. Just to clarify:
![image](https://user-images.githubusercontent.com/9705046/51927821-c1af6300-23f4-11e9-8ac3-ae8460a6fe1a.png)

Reply:
That's not how *any* of the other code works, or tools like the Object Editor for that matter. Orthogonal directions are NE/SE/SW/NW, diagonal directions are N/E/S/W.

Reply:
It is how footpath's edges and corners work, park fences, and probably more. I have not seen it the other way around, at least not for tile elements.